ccd-definition/ComplexTypes/CareSupervision/1_SupportingDocuments/SupportingEvidenceBundle.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "SupportingEvidenceBundle",
+    "ListElementCode": "name",
+    "FieldType": "Text",
+    "ElementLabel": "Document name",
+    "HintText": "TODO Try and make this more generic so we can reuse the same object",
```

Comment:
Do we still need this TODO?

Reply:
@Jamesnelson117 - You were looking at this - What do we do? Shall i leave it for now and you tackle in your PR?

Reply:
TODO was just a placeholder in my FPLA-2001. Remove.